IBPSA/Fluid/Actuators/Valves/TwoWayButterfly.mo

```diff
@@ -0,0 +1,30 @@
+within IBPSA.Fluid.Actuators.Valves;
+model TwoWayButterfly
+  "Two way valve with the flow characteristic of a typical butterfly valve"
+  extends IBPSA.Fluid.Actuators.Valves.TwoWayPolynomial(
+    final CvData = IBPSA.Fluid.Types.CvTypes.Kv,
+    Kv = 0.0472*(DN*1000)^2 -2.0104*(DN*1000),
+    final c={0,1.101898284705380E-01, 2.217227395456580,  - 7.483401207660790, 1.277617623360130E+01, -6.618045307070130});
+
+  parameter Modelica.SIunits.Length DN(displayUnit="mm") "Nominal diameter";
```

Comment:
This should be `d` for inner diameter. DN is usually based on outer diameter and not a good value to compute the inner diameter because the wall thickness depends on the material and rated pressure. Maybe we could add a table in the info section that shows typical inner diameters for a given DN.

Also, I suggest to not set the `displayUnit` attribute because in `HydaulicDiameter`, it is also not set. Then the parameter window will be consistent for the two models.

Reply:
We could use the inner diameter, but the source table that I used lists DN's. Do we keep the same data?

Reply:
I see. The data fit is based on an "inner diameter" which is set to be the same number as the DN value, even though they are typically not the same. I suggest to then just use a `choices` selecting that allows the user to select a DN value, and assigns the `Kv` value. Please see https://github.com/ibpsa/modelica-ibpsa/pull/1441/commits/50f11fc6554e6f0a27203369c410d35cee4adcf9